src/rshim.c

```diff
@@ -29,18 +29,54 @@
 /* RShim timer interval in milliseconds. */
 #define RSHIM_TIMER_INTERVAL 1
 
+/* Intervals to check the locked mode. */
+#define CHECK_LOCKED_MODE_MS      100
+#define CHECK_LOCKED_MODE_TICKS   (CHECK_LOCKED_MODE_MS / RSHIM_TIMER_INTERVAL)
+
+/* Timeouts in ms */
+#define TO_RSHIM_REQ_MS 5000
+#define TO_RSHIM_ONLINE_MS 5000
+
+/* Timeouts in ticks */
+#define TO_TICKS_RSHIM_REQ (TO_RSHIM_REQ_MS / OSP_MGT_INTERVAL_MS)
+#define TO_TICKS_RSHIM_ONLINE (TO_RSHIM_ONLINE_MS / OSP_MGT_INTERVAL_MS)
```

Comment:
I couldn't find where these 4 MACROs are used... 
If needed, we could consider some meaningful name with RSHIM_XXX prefix.

Reply:
The two 5 seconds are to define protocol time for the ownership requester waiting for ACK, and for request recipient to waiting for the requestor to be active. Right now it's hard coded to 1 second. Will change this value and the protocol code to reflect this.